index-alphabetic-browse.bat

```diff
@@ -43,8 +43,29 @@ goto javaset
 set JAVA="%JAVA_HOME%\bin\java"
 :javaset
 
+rem This would be /opt/bitnami/solr on a solr docker container
```

Comment:
I think it would be better to say this in a more generic way like "This can point to an external Solr in e.g. a Docker container" (also avoids documenting a current practice of a single vendor).